schemas/mw-page-delete.yaml

```diff
@@ -0,0 +1,53 @@
+title: MediaWiki Page Delete
+description: Represents a MW Page Delete event
+type: object
+properties:
+  # global event fields
+  meta:
```

Comment:
Re: `envelope`.  I prefer `meta`, but not strongly.  There is more stuff in here than just sender/receiver info, and there may be more added later (e.g. client_ip).  There also may be use cases where different types of meta data are used for different schemas (as long as some required fields always exist).  And!  I already did a bunch of code using the concept of 'meta', BUUUUT that isn't THAT hard to change ;)